addons/l10n_ch/i18n_extra/de.po

```diff
@@ -40,27 +38,32 @@ msgstr "0% Exkl."
 #. module: l10n_ch
 #: model:account.tax.template,description:l10n_ch.vat_O_import
 msgid "0% import."
-msgstr "0% Import."
+msgstr "VST 0% Import."
 
 #. module: l10n_ch
 #: model:account.tax.template,description:l10n_ch.vat_other_movements_900
 msgid "0% subventions"
-msgstr ""
+msgstr "0% Subv."
 
 #. module: l10n_ch
 #: model:account.tax.template,name:l10n_ch.vat_100_import
-msgid "100% dédouanement TVA"
-msgstr "100% Verzollung MwSt."
+msgid "Dédouanement TVA (biens et services)"
+msgstr "ZAZ 100% Mat.-/DL (OZD)"
 
 #. module: l10n_ch
 #: model:account.tax.template,description:l10n_ch.vat_100_import
 msgid "100% imp."
-msgstr "100% Imp."
+msgstr "ZAZ 100% Mat"
+
+#. module: l10n_ch
+#: model:account.tax.template,description:l10n_ch.vat_100_import_invest
+msgid "100% imp.invest."
+msgstr ""
 
 #. module: l10n_ch
 #: model:account.tax.template,description:l10n_ch.vat_25
 msgid "2.5%"
-msgstr "2.5%"
+msgstr "0.025"
```

Comment:
@rgo-odoo This is still weird

Reply:
@jco-odoo actually it was proposed to be changed like that, see column `7M` of `TAX` sheet from task pad.

Reply:
@rgo-odoo That is an error.  (probably to do with the spreadsheet application)

Reply:
2.5 %  is definitely easier to read.